docs/02-requirements/LZ-02-01.adoc

```diff
@@ -7,44 +7,38 @@ Architekten können Stakeholder und deren Anliegen sowie deren
 Auswirkungen auf die Softwarearchitektur oder den Entwurfs- und
 Entwicklungsprozess identifizieren. (R2)
 
-* Beispiele für Stakeholder und ihre Anliegen sind wie folgt:
+* Beispiele für Stakeholder und ihre Anliegen:
 
 |===
 |Interessengruppe |Anliegen der Interessengruppe
 
 | Produktmanagement
 | z. B. benötigte Zeit für die Umsetzung der Anforderungen
 
-| Projektmanager
+| Projektmanagement
 | z. B. benötigte Zeit und Budget für die Umsetzung, verbundene Risiken des
 gewählten architekturellen Ansatzes
 
-| Requirements Ingenieure
+| Requirements Engineering, Fachbereiche
 | z. B. Erfüllung der Anforderungen
 
-| Entwickler
+| Entwicklungsteam
 | z. B. zu implementierende Komponenten und Schnittstellen, Protokolle,
 technologische Einschränkungen
 
-| Qualitätssicherung und Tester
+| Qualitätssicherung und Test
```

Comment:
"Test" sounds a bit weird for me. What about "Test-Management" or "Testen"

Reply:
Testmanagement would be ok for me.

Reply:
As a role "Qualitätssicherung und Test" sounds OK to me and I like the short version  better than "Testmanagement" (which might confuse some people who might start wondering whether there is a difference to the role of testers).